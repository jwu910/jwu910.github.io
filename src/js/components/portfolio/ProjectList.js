import React from 'react';

import ProjectCard from './ProjectCard';

class ProjectList extends React.Component {
  render() {
    return (
      <div className="project-list-container">
        <ul>
          <li>
            <ProjectCard
              title="Check It Out"
              githubLink="/jwu910/check-it-out"
            />
          </li>

          <li>
            <ProjectCard
              title="Branchnote"
              githubLink="/jwu910/branchnote"
            />
          </li>

          <li>
            <ProjectCard
              title="Slng"
              githubLink="/5-gwoap/slng-node"
            />
          </li>
        </ul>
      </div>
    );
  }
}

export default ProjectList;