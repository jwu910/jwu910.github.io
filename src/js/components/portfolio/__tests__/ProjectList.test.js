import React from 'react';
import renderer from 'react-test-renderer';

import ProjectList from '../ProjectList';

describe('Snapshots', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(
        <ProjectList />
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});