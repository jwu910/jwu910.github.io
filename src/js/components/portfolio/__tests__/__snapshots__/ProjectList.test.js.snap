// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Renders correctly 1`] = `
<div
  className="project-list-container"
>
  <ul>
    <li>
      <div
        className="project-card-container"
      >
        <div
          className="title"
        >
          check-it-out
        </div>
      </div>
    </li>
    <li>
      <div
        className="project-card-container"
      >
        <div
          className="title"
        >
          branch-note
        </div>
      </div>
    </li>
    <li>
      <div
        className="project-card-container"
      >
        <div
          className="title"
        >
          slng
        </div>
      </div>
    </li>
  </ul>
</div>
`;
